packages/@aws-cdk/aws-glue/README.md

```diff
@@ -69,6 +69,33 @@ new glue.Table(stack, 'MyTable', {
 });
 ```
 
+### [Encryption](https://docs.aws.amazon.com/athena/latest/ug/encryption.html)
+
+You can enable encryption on a S3 bucket:
+* [SSE-S3](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html) - Server side encryption (SSE) with an Amazon S3-managed key.
+```ts
+new glue.Table(stack, 'MyTable', {
+  encryption: glue.TableEncryption.SSE_S3
```

Comment:
Enum names should be consistent with [`BucketEncryption`](https://awslabs.github.io/aws-cdk/refs/_aws-cdk_aws-s3.html#encryption)

Reply:
I would argue the other way around - the enum values are consistent with the S3, Athena, Glue and EMR documentation. What would I name `CSE-KMS` if I were copying `BucketEncryption`?

Reply:
Fair enough, but I think we have a problem with ALL_CAPS when converting those member names to other languages. Can we find names that are PascalCase?